tensorflow/lite/kernels/internal/tensor_utils.h

```diff
@@ -17,9 +17,11 @@ limitations under the License.
 
 #include <algorithm>
 #include <cmath>
+#include <cstdint>
 
-#include "third_party/eigen3/Eigen/Core"
 #include "tensorflow/lite/c/builtin_op_data.h"
+#include "tensorflow/lite/kernels/internal/tensor_utils_common.h"
+#include "third_party/eigen3/Eigen/Core"
```

Comment:
Include order needs to be as the original for Eigen/Core.

Reply:
Changed include file order.  Still fails the TFLite Makefile internal test.